cypress/support/server.js

```diff
@@ -0,0 +1,425 @@
+        memberIds = [memberIds];
+      }
+      const allMembers = memberIds?.map((id) => getMemberById(members, id));
+      // member does not exist in db
+      if (!allMembers) {
+        return reply({
+          statusCode: StatusCodes.NOT_FOUND,
+        });
+      }
+
+      return reply({
+        body: allMembers,
+        statusCode: StatusCodes.OK,
+      });
+    },
+  ).as('getMembers');
+};
+
+export const mockSignInRedirection = () => {
+  cy.intercept(
+    {
+      method: DEFAULT_GET.method,
+      url: `${AUTHENTICATION_HOST}/${buildSignInPath()}`,
+    },
+    ({ reply }) => {
+      reply(redirectionReply);
+    },
+  ).as('signInRedirection');
+};
+
+export const mockSignOut = () => {
+  cy.intercept(
+    {
+      method: DEFAULT_GET.method,
+      url: new RegExp(SIGN_OUT_ROUTE),
+    },
+    ({ reply }) => {
+      reply(redirectionReply);
+    },
+  ).as('signOut');
+};
+
+export const mockGetCategoryTypes = (categoryTypes) => {
+  cy.intercept(
+    {
+      method: DEFAULT_GET.method,
+      url: new RegExp(
+        `${API_HOST}/${parseStringToRegExp(GET_CATEGORY_TYPES_ROUTE)}$`,
+      ),
+    },
+    ({ reply }) => {
+      reply(categoryTypes);
+    },
+  ).as('getCategoryTypes');
+};
+
+export const mockGetCategories = (categories, shouldThrowError) => {
+  cy.intercept(
+    {
+      method: DEFAULT_GET.method,
+      url: new RegExp(
+        `${API_HOST}/${parseStringToRegExp(buildGetCategoriesRoute())}`,
+      ),
+    },
+    ({ reply }) => {
+      if (shouldThrowError) {
+        reply({ statusCode: StatusCodes.BAD_REQUEST, body: null });
+        return;
+      }
+      reply(categories);
+    },
+  ).as('getCategories');
+};
+
+export const mockGetItemCategories = (items, shouldThrowError) => {
+  cy.intercept(
+    {
+      method: DEFAULT_GET.method,
+      url: new RegExp(
+        `${API_HOST}/${parseStringToRegExp(
+          buildGetItemCategoriesRoute(items[0]?.id),
+        )}`,
+      ),
+    },
+    ({ reply, url }) => {
+      if (shouldThrowError) {
+        reply({ statusCode: StatusCodes.BAD_REQUEST, body: null });
+        return;
+      }
+      const itemId = url.slice(API_HOST.length).split('/')[2];
+      const result = items.find(({ id }) => id === itemId)?.categories || [];
+      reply(result);
+    },
+  ).as('getItemCategories');
+};
+
+export const mockGetItemsInCategories = (
+  items,
+  categories,
+  shouldThrowError,
+) => {
+  cy.intercept(
+    {
+      method: DEFAULT_GET.method,
+      url: new RegExp(
+        parseStringToRegExp(
+          `${API_HOST}/p/${ITEMS_ROUTE}/with-categories?${qs.stringify(
+            { category: [categories[0]?.id] },
+            { arrayFormat: 'repeat' },
+          )}`,
+        ),
+      ),
+    },
+    ({ reply }) => {
+      if (shouldThrowError) {
+        reply({ statusCode: StatusCodes.BAD_REQUEST, body: null });
+        return;
+      }
+      reply([items[0]]);
+    },
+  ).as('getItemCategories');
```

Comment:
Two problems here:
- the alias is the same as the one above
- maybe it's interesting to actually find the correct categories given `items`. Otherwise your browse page won't work at all.